### PR TITLE
Add definition for GuildChannel Parent

### PR DESCRIPTION
**Pull request by kzay**
Please describe the changes this PR makes and why it should be merged:

 This PR add the definition to set parent on GuildChannel

Semantic versioning classification:

 This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by zajrik** [comment]
The `#region` comments should be left unindented. They are this way to allow folding to collapse regions when working on other sections of the file in editors that support arbitrary block folding. They will not function if they are indented to the same level as the enclosed code.

**Comment by kzay** [comment]
Oh! Ok i revert the identation, it was my auto formatting tool.

File: index.d.ts [hunk 1]
```diff
@@ -1600,25 +1601,25 @@ declare module 'discord.js' {
 		maxID?: Snowflake;
 		minID?: Snowflake;
 		has?: 'link'
-			| 'embed'
```
Comment (zajrik) [inline index.d.ts]: I'm not sure why you changed the indentations on these, but this should be reverted. I'll assume this was also your formatting tool. You should avoid using that when contributing to someone else's projects.

**Review by zajrik — CHANGES_REQUESTED** [review]

**Review by kzay — COMMENTED** [review]
Yes TS formatter, but i agreed

File: index.d.ts (hunk 1 above)
Comment (kzay) [reply index.d.ts]: Its done.

**Pull request merged**